store/homeSection.ts

```diff
@@ -10,7 +10,7 @@ import { AppState } from './index';
 
 export interface HomeSection {
   name: string;
-  libraryName: string | undefined;
+  libraryName?: string | null | undefined;
```

Comment:
What does `?` add more than the union?

Reply:
In an interface, it marks the property as optional.